custom_schemas/custom_process.yml

```
@@ -188,6 +188,14 @@
       description: The sha256 of the call_stack_final_user_module.
       example: "d25ff1e6c6460a7f9de39198d182058c1712726008d187e1953b83abe977e4a0"
 
+    - name: thread.Ext.hardware_breakpoint_set
+      level: custom
+      type: boolean
+      description:  >
+        Check whether a hardware breakpoint is set for the thread. 
+        Include this field, only if the hardware breakpoint is setted. 
+      example: "true" 
```

This generates a non-fatal warning - "Short descriptions must be single line, and under 120 characters"
You can explicitly set a short description with `short:`

```suggestion
      short: Whether a hardware breakpoint was set for the thread. 
      description:  >
        Whether a hardware breakpoint was set for the thread. 

        This field is omitted if false.
      example: "true" 
```

---

Thanks! Fixed as suggested!